client/src/pages/onboarding/steps/PickPersona.tsx

```diff
@@ -27,6 +28,7 @@ export const PickPersona: React.FC<Props> = ({ currentPersona, personas, title,
 
   const PersonaClickHandler = (char: CustomCharacter) => {
     dispatch(setPersona(char))
+    setOnboardingProgress(dispatch, 1)
```

Comment:
Please add FIXME to make this configurable or put this behind a callback. Maybe we want a pick persona in another part of another flow.